lib/code_corps/cloudex/cloudinary_url.ex

```diff
@@ -7,5 +7,11 @@ defmodule CodeCorps.Cloudex.CloudinaryUrl do
   end
   def for(public_id, options, _version, _default_color, _type) do
     @cloudex.Url.for(public_id, options)
+    |> add_html_scheme
+  end
+
+  def add_html_scheme(generated_url) do
+    base_url =  String.split(generated_url, "//")
+    if (length(base_url) > 0), do: "https://" <> List.last(base_url), else: generated_url
   end
```

Comment:
Could be rewritten without an iff using something like

```Elixir
defp add_html_scheme("//" <> base_url), do: "https://" <> base_url
defp add_html_scheme(url), do: url
```

Note that I also made the functions private, since they're not being called externally.